hadoop/src/main/java/org/apache/carbondata/hadoop/testutil/StoreCreator.java

```diff
@@ -407,6 +407,7 @@ public static void loadData(CarbonLoadModel loadModel, String storeLocation)
     CSVInputFormat.setNumberOfColumns(
         configuration, String.valueOf(loadModel.getCsvHeaderColumns().length));
     CSVInputFormat.setMaxColumns(configuration, "10");
+    CSVInputFormat.setLineSeparator(configuration, loadModel.getLineSeparator());
```

Comment:
If not null, then only set the line separator. Else it may lead to wrong behavior.
You already did like this in `CommonUtil.scala`
Follow the same

Reply:
> If not null, then only set the line separator. Else it may lead to wrong behavior.
> You already did like this in `CommonUtil.scala`
> Follow the same

`public static void setLineSeparator(Configuration configuration, String lineSeparator) {
    if (lineSeparator != null && !lineSeparator.isEmpty()) {
      configuration.set(LINE_SEPARATOR, lineSeparator);
    }
  }`
This is done inside the setLineSeparator.